src/server/serve.rs

```diff
@@ -72,6 +72,13 @@ fn core_serve_loop(_config_path: &str, port: u32) -> Result<(), std::io::Error>
             let mut thread_handles = vec![];
             let kill_switch = Arc::new(Mutex::new(false));
 
+            // kill server if listen_ip's are empty
+            if listen_ip_v4.is_empty() && listen_ip_v6.is_empty() {
+                //no configuration found kill the server
+                debug!("No configuration found stopping the sever");
```

Comment:
Can we do this at a level that is visible by default.

Reply:
@Luap99 I am sorry but I did not understand what do we mean by `visible` by default. Could you help me understand it.

We want to stop aardvark when no listen ip's are present. That is the intention of this PR.

Reply:
Did you mean make it configurable ?

Reply:
debug lines are not printed unless debug level is set, I want to see this line by default (when no log level is explicitly set). Just to make sure it exited intentionally and did not crashed for some other reason.